flowmachine/flowmachine/core/server/query_schemas/joined_spatial_aggregate.py

```diff
@@ -47,9 +49,32 @@ class JoinedSpatialAggregateSchema(Schema):
     query_kind = fields.String(validate=OneOf(["joined_spatial_aggregate"]))
     locations = fields.Nested(InputToSpatialAggregate, required=True)
     metric = fields.Nested(JoinableMetrics, required=True)
-    method = fields.String(
-        default="mean", validate=OneOf(JoinedSpatialAggregate.allowed_methods)
-    )
+    method = fields.String(validate=OneOf(JoinedSpatialAggregate.allowed_methods))
+
+    @pre_load
+    def validate_method(self, data, **kwargs):
+        quant_metrics = [
+            "radius_of_gyration",
+            "unique_location_counts",
+            "topup_balance",
+            "subscriber_degree",
+            "topup_amount",
+            "event_count",
+            "nocturnal_events",
+        ]
+        qual_metrics = ["handset"]
```

Comment:
See comment above. I feel it would help to rename these variables to reflect the continuous vs. categorical distinction as opposed to qualitative/quantitative (because even the categorical metrics still return a quantitative result).